src/utils/AnimalDomainHelper/AnimalDomainHelper.ts

```diff
@@ -32,13 +36,17 @@ export default class AnimalDomainHelper {
     const attributes: OpenSeaMetadataAttribute[] = [];
     const { prefix, animal } = this.extractPrefixAndAnimal(name);
     if (prefix && AdjectivesDictionary.includes(prefix)) {
-      attributes.push({ trait_type: 'adjective', value: prefix });
+      attributes.push({ trait_type: DomainTraitType.Adjective, value: prefix });
     }
     if (animal) {
-      attributes.push({ trait_type: 'animal', value: animal });
+      attributes.push({ trait_type: DomainTraitType.Animal, value: animal });
     }
     return attributes;
   }
+  isAnimalDomain(domainName: string): boolean {
```

Comment:
Nitpick: would be great to have a newline in front of the function for better readability.
```suggestion

  isAnimalDomain(domainName: string): boolean {
```